tests/test_meta_tag_hidden.py

```diff
@@ -0,0 +1,118 @@
+from tests.pyxform_test_case import PyxformTestCase
+
+"""
+Test hidden meta tags functionality in pyxform.
+Tests that meta elements get the tag="hidden" attribute for CHT SMS functionality.
+"""
+
+
+class MetaTagHiddenTest(PyxformTestCase):
+    """Test cases for hidden meta tags."""
+
+    def test_meta_tag_hidden_by_default(self):
+        """Test that meta elements get tag='hidden' attribute by default."""
+        self.assertPyxformXform(
+            name="test_meta",
+            md="""
+            | survey |        |         |       |
+            |        | type   | name    | label |
+            |        | text   | q1      | Q1    |
+            """,
+            xml__contains=[
+                '<meta tag="hidden">',
+                "<instanceID/>",
+                "</meta>",
+            ],
```

Comment:
I don't think this `xml__contains` asserts any specific order/orientation for the strings.  So, it think it would be a little stricter if we just had:

```suggestion
            xml__contains=[
                '<q1/>',
                '<meta tag="hidden">\n            <instanceID/>'
            ],
```